### DISCUSSION TITLE
Livewire 3 cannot use emit from Javascript

### DISCUSSION THREAD

**Discussion by alqaj** [livewire/livewire#7752] [1. Help]
I use Livewire 3 to call a Livewire Component  function from Javascript.
in Livewire 2, I could use window.livewire.emit
But, not for livewire 3.
How do that with dispatch or other from JS?
THanks

**Comment by danie-ramdhani** [comment]
try `dispatch` : https://livewire.laravel.com/docs/upgrading#events

**Reply by alqaj** [reply]
dispatch is function for calling inter Livewire Component. But cannot call from Javascript/ :(

btw, thank you

**Comment by kgsint** [comment]
@alqaj  You can do so with Livewire global object 
```javascript 
// Dispatch an event to any Livewire components listening...
Livewire.dispatch('post-created', { postId: 2 })
 
// Dispatch an event to a given Livewire component by name...
Livewire.dispatchTo('dashboard', 'post-created', { postId: 2 })
 
// Listen for events dispatched from Livewire components...
Livewire.on('post-created', ({ postId }) => {
    // ...
})
```
Ref: [https://livewire.laravel.com/docs/javascript#interacting-with-events](https://livewire.laravel.com/docs/javascript#interacting-with-events)
